installation/release-notes-main/release-notes-9-0-1.adoc

```diff
@@ -0,0 +1,54 @@
+= {PRODUCT_LONG}, 9.0.1, release notes
```

Comment:
We'll need to include the attributes above the title, or write the whole name here.